### PR TITLE
Custom featurizing

### PR DESCRIPTION
**Pull request by John-HarringtonNZ**

### CODE REVIEW

**Comment by mesutyang97** [comment]
Hi John,

We received your pull request, and thanks for the contribution! 

Could you please update the PR description to include a breif summary of what you changed and why? 

I am going to leave some comments for some specific areas that I am unsure about.

Thank you again for all the hard work you put into improving the framework.

Best,
Mesut

File: src/overcooked_ai_py/mdp/overcooked_mdp.py [hunk 1]
```diff
@@ -753,8 +753,8 @@ def __hash__(self):
         )
 
     def __str__(self):
-        return 'Players: {}, Objects: {}, Bonus orders: {} All orders: {} Timestep: {}'.format( 
-            str(self.players), str(list(self.objects.values())), str(self.bonus_orders), str(self.all_orders), str(self.timestep))
+        return 'Players: {}, Objects: {}'.format(
```
Comment (mesutyang97) [inline src/overcooked_ai_py/mdp/overcooked_mdp.py:756]: I believe bonus orders are still being used by parts of the environment, and we would appreciate if you could allow for this flexibility.

File: src/overcooked_ai_py/mdp/overcooked_env.py [hunk 2]
```diff
@@ -625,4 +634,8 @@ def reset(self):
                 "other_agent_env_idx": 1 - self.agent_idx}
 
     def render(self, mode="human", close=False):
-        pass
+        import pygame
```
Comment (mesutyang97) [inline src/overcooked_ai_py/mdp/overcooked_env.py:637]: Thanks for adding this!

**Comment by John-HarringtonNZ** [comment]
This is still a work in progress, and to be honest, the pull request was meant for my dev fork. Sorry for the mistake, and I'd be happy to put in another completed PR when it's in a more complete state! 

Best,
John

**Pull request closed without merging**